src/routes/AlwaysAccesibleRoute.tsx

```diff
@@ -13,6 +14,11 @@ interface Props extends RouteProps {
   routeProps?: RouteComponentProps | routePropsFn;
 }
 
+// Initiate Userflow
+if (process.env.USERFLOW_TOKEN) {
+  userflow.init(process.env.USERFLOW_TOKEN);
+}
```

Comment:
Why put this in the route and not inside the app index? _(as that is the initial entry point)_

Reply:
I originally had it in the root index.ts, however, thought it was matching the pattern of the app by putting it alongside FeedbackWidget and BetaCautionAlert components.

I will move it into the index.ts though, seeing as though you raised it.